db/migrate/20220928193827_add_online_to_room.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddOnlineToRoom < ActiveRecord::Migration[7.0]
+  def change
+    add_column :rooms, :online, :boolean
```

Comment:
What was the deciding factor for creating a column for `online` vs the approach you had before?

Reply:
It's to avoid doing an expensive `active_meetings` query on every Rooms refresh. We store the meeting status in the database and then we double check only the online meetings status with `get_meeting_info`